### PR TITLE
New package: yEd-3.19.1.1_1

### PR DESCRIPTION
**Pull request by jbenden**
Signed-off-by: Joseph Benden <joe@benden.us>

### CODE REVIEW

**Comment by Anachron** [comment]
Shouldnt it be in nonfree?

**Comment by jbenden** [comment]
Yes. I have adjusted the template.

Thanks!!

File: srcpkgs/yed/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'yed'
+pkgname=yed
+version=3.19.1.1
+revision=1
+archs=noarch
+wrksrc="$pkgname-${version}"
+hostmakedepends="bsdtar unzip"
```
Comment (xtraeme) [inline srcpkgs/yed/template]: This is unnecessary

File: srcpkgs/yed/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+depends="virtual?java-environment"
+short_desc="Very powerful graph editor written in Java"
+maintainer="Joseph Benden <joe@benden.us>"
+license="custom:Commercial"
+homepage="https://www.yworks.com/products/yed/"
+distfiles="https://www.yworks.com/resources/yed/demo/yEd-${version}.zip"
+checksum="3639da2650976bffcd43a8ea830727bae86478e95ceb4d68ad6363a42a8f1efb"
+
+repository=nonfree
```
Comment (xtraeme) [inline srcpkgs/yed/template]: if we aren't allowed to distribute it, set `restricted=yes`

File: srcpkgs/yed/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'yed'
+pkgname=yed
+version=3.19.1.1
+revision=1
+archs=noarch
+wrksrc="$pkgname-${version}"
```
Comment (xtraeme) [inline srcpkgs/yed/template]: this is the default, unnecessary

File: srcpkgs/yed/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	vmkdir usr/share/yed
+	vmkdir usr/bin
+
+	vbin ${FILESDIR}/yed
+	vcopy lib usr/share/yed
+	vcopy yed.jar usr/share/yed
+	vcopy icons/yed64.png usr/share/pixmaps/yed.png
+	vcopy ${FILESDIR}/yed.desktop usr/share/applications/yed.desktop
+
+	vlicense license.html thirdpartylicenses.html
```
Comment (Chocimier) [inline srcpkgs/yed/template]: this install one file license.html under name thirdpartylicenses.html, is it what was meant?

File: srcpkgs/yed/template (hunk 1 above)
Comment (jbenden) [reply srcpkgs/yed/template]: Thank you!

File: srcpkgs/yed/template (hunk 2 above)
Comment (jbenden) [reply srcpkgs/yed/template]: Thank you... Apparently, still learning Void Linux ropes... :)

File: srcpkgs/yed/template (hunk 3 above)
Comment (jbenden) [reply srcpkgs/yed/template]: Thanks!

File: srcpkgs/yed/template (hunk 4 above)
Comment (jbenden) [reply srcpkgs/yed/template]: Good catch! That's not what I wanted.

Thank you!

**Comment by jbenden** [comment]
Does anyone know why this occurred: `=> ERROR: yed-3.19.1.1_1: cannot find unzip or bsdtar bin for extraction.`

It happened on musl, but not glibc.

**Comment by jbenden** [comment]
I added unzip to solve the musl build error.

**Pull request closed without merging**